lib/job-iteration.rb

```diff
@@ -9,6 +9,8 @@ module JobIteration
 
   INTEGRATIONS = [:resque, :sidekiq]
 
+  Deprecation = ActiveSupport::Deprecation.new("2.0", "JobIteration")
```

Comment:
I originally used `ActiveSupport::Deprecation.warn` directly, that labelled deprecations as coming from Rails. This uses our own object, but does mean that the host app's settings for `ActiveSupport::Deprecation` do not propagate (e.g. `disallowed_deprecations`).